go/vt/tabletmanager/gorpctmserver/gorpc_server.go

```diff
@@ -35,6 +36,7 @@ type TabletManager struct {
 
 // Ping wraps RPCAgent.
 func (tm *TabletManager) Ping(ctx context.Context, args, reply *string) error {
+	ctx = callinfo.RPCWrapCallInfo(ctx)
```

Comment:
I merged this, so I can run the import and fix all the associated files
this morning... I'll fix forward if I missed something.

On Mon, Mar 9, 2015 at 6:55 PM, sougou <notifications@github.com> wrote:

> In go/vt/tabletmanager/gorpctmserver/gorpc_server.go
> <https://github.com/youtube/vitess/pull/460#discussion_r26093923>:
>
> > @@ -35,6 +36,7 @@ type TabletManager struct {
> >
> >  // Ping wraps RPCAgent.
> >  func (tm *TabletManager) Ping(ctx context.Context, args, reply *string) error {
> > +	ctx = callinfo.RPCWrapCallInfo(ctx)
>
> Why is this needed only for TabletManager and not for others like
> SqlQuery, VTGate, etc?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/youtube/vitess/pull/460/files#r26093923>.
>